packages/playground/src/components/node_selection.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <v-card color="primary" class="d-flex mt-3 pa-3">
+    <v-card-title class="pa-0">Node Selection</v-card-title>
+  </v-card>
+  <h6 class="text-h6 mb-4 mt-2 ml-2">Choose a way to select Node</h6>
+  <v-radio-group v-model="selection" color="primary" inline>
+    <v-radio label="Automated" value="automated"></v-radio>
+    <v-radio label="Manual" value="manual"></v-radio>
+  </v-radio-group>
+</template>
```

Comment:
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/3ced7938-277d-4837-a6d5-39cdbfbeaa27)

Suggested enhancements:
- remove Choose a way to select Node
- add some space between 2 options
- decrease card height